src/Hit/Git/Status.hs

```diff
@@ -136,6 +137,34 @@ parseDiffStat = \case
         }
     _ -> Nothing
 
+{- | Attempts to expand shortened paths which can appear in `git diff --stat`.
+Examples of possible paths:
+
+@
+ a.in      => b.out
+ test/{bar => baz}
+ test/{bar => a1{/baz}
```

Comment:
Curly braces in this particular example are unbalanced. Is this a typo or `git` outputs in the same format?

Reply:
Yeah, that's what git showed, when the folder was called `a1{`.